packages/@atjson/offset-annotations/src/annotations/cne-audio-embed.ts

```diff
@@ -0,0 +1,21 @@
+import { BlockAnnotation } from "@atjson/document";
+
+export enum AudioEnvironments {
+  Production = "production",
+  Sandbox = "sandbox",
+}
+
+export const CneAudioEnvironments = {
+  [AudioEnvironments.Production]: `https://embed-audio.cnevids.com`,
+  [AudioEnvironments.Sandbox]: `https://embed-audio-sandbox.cnevids.com`,
+};
```

Comment:
This should not be here. This is a client concern (for ex, in Verso, the production/sandbox hostname will be a tenant config)

Reply:
I don't have pretty clear the use, but it is used by:
packages/@atjson/renderer-html/src/index.ts
Do you think is better to add a fake domain there and remove this `CneAudioEnvironments`?

Reply:
Yes the renderer can determine the hostname to use there